base_layer/core/src/mempool/mempool.rs

```diff
@@ -112,7 +112,7 @@ where T: BlockchainBackend
     }
 
     fn check_timelocks(&self, tx: &Transaction) -> Result<bool, MempoolError> {
-        match tx.max_timelock_height() {
+        match tx.max_restriction_height() {
```

Comment:
The old name makes more sense since it's an industry term. restriction?

Reply:
I specifically changed the name away from the industry term as the function is not doing what it says. 

Timelock implies that is gets the max timelock as specified in the kernel. That function looks at the timelock in the kernel as well as the maturity counter in the utxo.  It does both. 

On a transaction you have 2 types of timelocks. Maturity which is mined, kernel which is not mined. They can be accessed by: `max_input_maturity(&self)` -and `max_kernel_timelock(&self)` . `max_restriction_height` just combines them into a single call

Reply:
Function is now called `min_spendable_height`